Assets/UI/WorldView/citybannermanager_CQUI.lua

```diff
@@ -874,6 +946,20 @@ function IsCQUI_SmartBanner_Unmanaged_CitizenEnabled()
     return (CQUI_SmartBanner and CQUI_SmartBanner_Unmanaged_Citizen);
 end
 
+-- ===========================================================================
+function CQUI_SetCityStrikeButtonLocation(cityBannerInstance, rotate, offsetY, anchor)
+    cityStrikeImage = nil;
+    if (IsExpansion1Active() or IsExpansion2Active()) then
```

Comment:
These functions call Modding.IsModActie every time they are used. Which is unnecessary because the mod cannot be disabled or enable while in the game. That is why I added global variables to CQUICommon to use instead. They initialized once and can be used at any time, no extra calls.

Reply:
oh, right!  forgot about those, will update before I complete the PR